components/alerts/alert.css

```diff
@@ -12,8 +12,22 @@
 .alert-error-btn {
   max-width: 25rem;
   border-radius: 4px;
-  margin: 1rem;
+  margin: 0.6rem;
   padding: 0.6rem 0.6rem 0.6rem 0.8rem;
+  display: flex;
+  align-items: center;
+}
+
+.alert-success,
+.alert-success-btn,
+.alert-info,
+.alert-info-btn,
+.alert-warning,
+.alert-warning-btn,
+.alert-error,
+.alert-error-btn{
+  min-width: 18rem;
+  height: auto;
 }
```

Comment:
Maybe create a class 'alert' and write all these properties into it.

Reply:
These properties are combined to reduce the classes.